src/EditorFeatures/TestUtilities/Remote/WatsonReporter.cs

```diff
@@ -1,23 +1,29 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
- 
- using System;
- 
- namespace Microsoft.VisualStudio.LanguageServices.Implementation
- {
-     /// <summary>
-     /// Mock to make test project build
-     /// </summary>
-     internal class WatsonReporter
-     {
-         public static void Report(string description, Exception exception, Func<IFaultUtility, int> callback)
-         {
-             // do nothing
-         }
- 
-         public interface IFaultUtility
-         {
-             void AddProcessDump(int pid);
-             void AddFile(string fullpathname);
-         }
-     }
- }
\ No newline at end of file
+
+using System;
```

Comment:
:bulb: In the future, can you submit formatting changes only in commits which contain *no other changes*? Ignoring in this case only because the file is so short.